### PR TITLE
Breaking: Replace `chrome` with `puppeteer`

### LINKED ISSUE #2254: Update `configuration` packages to use `connector-chromium`

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

I'm also making puppeteer the default connector here. This should hopefully give the user a better experience and less issues with brotli and canvas.

Fix #2254

### CODE REVIEW

File: packages/configuration-progressive-web-apps/index.json [hunk 1]
```diff
@@ -1,9 +1,6 @@
 {
     "connector": {
-        "name": "jsdom",
-        "options": {
-            "waitFor": 5000
-        }
+        "name": "puppeteer"
```
Comment (sarvaje) [inline packages/configuration-progressive-web-apps/index.json]: Very Nit: I thing this particular change should have its own commit

File: packages/configuration-web-recommended/index.json [hunk 2]
```diff
@@ -1,9 +1,6 @@
 {
     "connector": {
-        "name": "jsdom",
-        "options": {
-            "waitFor": 5000
-        }
+        "name": "puppeteer"
```
Comment (sarvaje) [inline packages/configuration-web-recommended/index.json]: Very Nit: I thing this particular change should have its own commit

Comment (molant) [reply packages/configuration-web-recommended/index.json]: I was debating that but it's 2 breaking changes in a row so might as well get it done in one. I could rename the message though to
>  Breaking: Replace `chrome` with `puppeteer` and make it the default

Comment (sarvaje) [reply packages/configuration-web-recommended/index.json]: Ok, that could work too :)

Comment (molant) [reply packages/configuration-web-recommended/index.json]: OK, I'll do it at merge time then :)

**Comment by molant** [comment]
Seems like everyone is happy so I'll merge with the message changed.

**Pull request merged**